packages/kcms/src/match/service/fetchRunResult.ts

```diff
@@ -0,0 +1,33 @@
+import { MatchType } from 'config';
+import { RunResult } from '../model/runResult';
+import { MainMatchID } from '../model/main';
+import { PreMatchID } from '../model/pre';
+import { Option, Result } from '@mikuroxina/mini-fn';
+import { MainMatchRepository, PreMatchRepository } from '../model/repository';
+
+export class FetchRunResultService {
+  constructor(
+    private readonly mainMatchRepository: MainMatchRepository,
+    private readonly preMatchRepository: PreMatchRepository
+  ) {}
+  async fetch(
```

Comment:
```suggestion
  async handle(
```
が望ましいようです(serviceにメソッドが1つしか存在しないため)。